src/__tests__/Scanner.test.ts

```diff
@@ -123,3 +123,43 @@ test('scanTokens ignores internal spaces', () => {
     },
   ])
 })
+
+test('scanTokens handles multicharacter strings', () => {
+  const tokens = new Scanner('10 + 18').scanTokens()
+  expect(tokens).toEqual([
+    {
+      ttype: TokenType.NUMBER,
+      text: '10',
+      literal: 10,
+    },
+    {
+      ttype: TokenType.PLUS,
+      text: '+',
+    },
+    {
+      ttype: TokenType.NUMBER,
+      text: '18',
+      literal: 18,
+    },
+  ])
+})
+
+test('scanTokens handles decimal numbers', () => {
```

Comment:
Let's add a test for a leading decimal point ie: `.25`

Reply:
Added a test to check for leading decimal points and multiple numbers with leading decimal points